raaghu-elements/src/rds-plan-switcher/rds-plan-switcher.tsx

```
@@ -0,0 +1,59 @@
+ 
+export interface RdsPlanSwitcherProps {
+  setIsPlanFree?: (isPlanFree: boolean) => void;
+  button1Text: string;
+  button2Text: string;
+}
+ 
+const RdsPlanSwitcher = (props:RdsPlanSwitcherProps) => {
+  const [isPlanFree, setIsPlanFree] = useState(true);
+
+  useEffect(() => {
+    props.setIsPlanFree && props.setIsPlanFree(isPlanFree);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)